src/Deserialization/Transaction.purs

```diff
@@ -416,21 +417,21 @@ convertMetadatum
 convertMetadatum err = fix \_ -> addErrTrace err <<< _convertMetadatum
   ( { error: fromCslRepError
     , from_bytes: pure <<< Bytes
-    , from_int: map Int <<< stringToBigInt "Metadatum Int"
+    , from_int: map Int <<< stringToInt "Metadatum Int"
     , from_text: pure <<< Text
     , from_map: convertMetadataMap convertMetadatum
     , from_list: convertMetadataList convertMetadatum
     }
   )
 
   where
-  stringToBigInt
+  stringToInt
     :: forall s
```

Comment:
:: forall (s :: Row Type)

Reply:
There's also another one in `Deserialization.Transaction` under `intToSlot` if you wouldn't mind changing that.